metashare/api/sf_org_changes.py

```diff
@@ -13,7 +13,60 @@
 from .sf_run_flow import refresh_access_token
 
 
-def run_retrieve_task(user, scratch_org, project_path, desired_changes):
+def get_valid_target_directories(scratch_org_config):
+    """
+    Expects to be called from within a `local_github_checkout`.
+    """
+    package_directories = {}
+    if scratch_org_config.get("source_format") == "sfdx":
+        with open("sfdx-project.json") as f:
+            sfdx_project = json.loads(f.readlines())
```

Comment:
Out of curiosity, what's the difference between `json.loads(f.readlines())` and `json.load(f)` used below?

Reply:
They amount to the same thing.